src/pages/index.tsx

```diff
@@ -61,6 +69,16 @@ const Home: NextPage<any> = () => {
       setMapState(newMap);
     }
   }, [user, geoJsonUsers]);
+
+  const addFavorite = (favoriteId: string, add: boolean) => {
```

Comment:
sorry to be nitpicky but can we rename this to handleFavorite just for readability?